types/yargs/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for yargs 15.0
+// Type definitions for yargs 16.0
 // Project: https://github.com/chevex/yargs, https://yargs.js.org
```

Comment:
There is no change between v16 and v15 currently. This change needs to be hold off if there is no change between these two versions

Reply:
`strictOptions` [doesn't exist in v15](https://github.com/yargs/yargs/compare/v15.4.2...v16.0.0#diff-452e4f3371a75b323f89da8de1a1707b1532abef67be1a02fe7e8396e0f35020R955). That's the point of this change.

Reply:
I dont see any change in v16? Can you point where the change is between v15 and v16 in these definitions?

Reply:
`.strictOptions` [exists here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/72cb30219ded5aa79b9fa554463f8f860e2e0f08/types/yargs/index.d.ts#L579) and i've [removed it here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/72cb30219ded5aa79b9fa554463f8f860e2e0f08/types/yargs/v15/index.d.ts#L573). See the [diff](https://diffy.org/diff/nngnb82nreufkht6njdc84cxr)

Reply:
Thank you. it got lost in big copy.. normally when versions change the change is in index.d.ts and not the older version. Thank you for the pointers.